### PR TITLE
Details popup

### PR DESCRIPTION
**Pull request by Nedramrez**
- Prepare the javascript code
- Add mobile version details popup window
- Add desktop version details popup window

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -7,3 +7,50 @@ function mobileMenu() {
...
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    seeLive: 'https://nedramrez.github.io/portfolio/',
+    seeSource: 'https://github.com/nedramrez/portfolio',
+  },
+};
```
Comment (omarsalem7) [inline script.js]: - [ ] Kindly consider making the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads).
- [ ] Also consider modal popup data is populated with the data from the projects array.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Nedramrez 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -7,3 +7,114 @@ function mobileMenu() {
...
+      <div class="action">
+        <div class="see-project">
+          <button onclick="detailsPp()" class="see-pp">See Project</button>
+        </div>
+      </div>
+    </div>
+  `;
+  portfolio.appendChild(cardWorks);
+});
+
```
Comment (zieeco) [inline script.js:111]: **[REQUIRED CHANGES]**

- [ ] Hey, great job with the implementation, however, there are some issues to be addressed. Please, kindly fix the following issues stated below:

- The project images don't show on mobile and desktop versions of your portfolio website but only show on the modal pop-up.
- The mobile version of the modal doesn't show all the content. The `see live` and `see source` buttons are not showing up.
- The desktop version of the modal doesn't match the Figma design:

- [x]  The cancel button is not showing.
- [ ] The image is not well aligned as that on the Figma design.
- [x] The `see live` and `see source` buttons are not showing up.

:point_right: **Please, kindly refer to the Figma design to fix the issue**

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Nedramrez,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions

- [ ] Please do try as much as possible to ensure your design doesn't have a horizontal scroll behavior to improve the user experience.

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (Nedramrez) [reply script.js:111]: Hi @zieeco !
Thank you for your professional and friendly review.
Please, help me to understand the required changes. 
I cloned my repository to a different folder to check again and see all project images, the cancel button of the modal, all buttons showing up etc.
I attached screenshots.
Скриншот, уақыты 2022-06-22 20-56-19

Скриншот, уақыты 2022-06-22 20-54-14
Скриншот, уақыты 2022-06-22 20-54-29
Скриншот, уақыты 2022-06-22 20-54-37
Скриншот, уақыты 2022-06-22 20-54-58

Comment (Nedramrez) [reply script.js:111]: And I asked my partner to clone. He shared his screen and he could see all images and buttons too.

Comment (zieeco) [reply script.js:111]: @Nedramrez, It shows well locally, but not online I mean, kindly change the github pages to that of this detailed pop branch so I check again.

Comment (zieeco) [reply script.js:111]: **[REQUIRED CHANGES]**

- Hey, while every other change has been met, there's still an issue with the image in your pop up, see the video below and compare with the Figma design

*your pop p in action*

https://user-images.githubusercontent.com/59397159/175076569-6b507122-0862-4818-a63d-d05cef7d1e75.mp4

*figma design*
figma__

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Nedramrez,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- [ ] Please do try as much as possible to ensure your design doesn't have a horizontal scroll behavior to improve the user experience.


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Nedramrez,

YesYouAreGIF

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights

✅ Correct use of Gitflow
✅ No linter error

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Nedramrez** [comment]
Thank you very much!

**Pull request merged**